Added Project Monitors page for 2.6.7 (#4222)
rancher/docs: content/rancher/v2.6/en/monitoring-alerting/prometheus-federator/guides/enable-prom-fed/_index.md

## Ensure the cattle-monitoring-system namespace is placed into the System Project (or a similarly locked down Project that has access to other Projects in the cluster)

![Select Projects-Namespaces]({{<baseurl>}}/img/rancher/install-in-system-project.png)

Prometheus Operator's security model expects that the namespace it is deployed into (e.g., `cattle-monitoring-system`) has limited access for anyone except Cluster Admins to avoid privilege escalation via execing into Pods (such as the Jobs executing Helm operations). In addition, deploying Prometheus Federator and all Project Prometheus stacks into the System Project ensures that each Project Prometheus is able to reach out to scrape workloads across all Projects, even if Network Policies are defined via Project Network Isolation. It also provides limited access for Project Owners, Project Members, and other users so that they're unable to access data that they shouldn't have access to (i.e., being allowed to exec into pods, set up the ability to scrape namespaces outside of a given Project, etc.).

> 1. Open the `system` project to check your namespaces:

Reviewer: ```suggestion
1. Open the `System` project to check your namespaces:
```

Author: Updated as suggested.

>     Click **Cluster > Projects/Namespaces** in the Rancher UI. This will display all of the namespaces in the `system` project:

Reviewer: ```suggestion
    Click **Cluster > Projects/Namespaces** in the Rancher UI. This will display all of the namespaces in the `System` project:
```

Author: Updated as suggested.

    ![Select Projects-Namespaces]({{<baseurl>}}/img/rancher/cattle-monitoring-system.png)

> 1. If you have an existing monitoring V2 installation in which `cattle-monitoring-system` is not in the `system` project, you may move the `cattle-monitoring-system` namespace into the `system` project or into another project of limited access. To do so, you may either:

Reviewer: ```suggestion
1. If you have an existing Monitoring V2 installation within the `cattle-monitoring-system` namespace but that namespace is not in the `System` project, you may move the `cattle-monitoring-system` namespace into the `System` project or into another project of limited access. To do so, you may either:
```

Author: Updated as suggested.

>     - Drag and drop the namespace into the `system` project or

Reviewer: ```suggestion
    - Drag and drop the namespace into the `System` project or
```

Author: Updated as suggested.

>     - Select **⋮** to the right of the namespace, click **Move**, then choose `System` from the **Target Project** dropdown

Reviewer: is the `**:**` intentional?

Author: Yes, I was bolding out the vertical ellipses: **⋮**.
